bundles/pom.xml

```diff
@@ -47,6 +47,7 @@
     <module>org.openhab.binding.amazondashbutton</module>
     <module>org.openhab.binding.amazonechocontrol</module>
     <module>org.openhab.binding.ambientweather</module>
+    <module>org.openhab.binding.androiddebugbridge</module>
```

Comment:
Can you revert these changes?

Reply:
Created a new fork, this should be back OK